Makefile

```diff
@@ -67,16 +69,30 @@ endif
 ifdef S3_SELF_SIGNED_CERT
 	oc create configmap s3-self-signed-cert --from-file=ca-bundle.crt=${S3_SELF_SIGNED_CERT}
 endif
+ifndef GIT_TOKEN
+	$(error GIT_TOKEN is undefined)
+endif
+ifndef GIT_USERNAME
+	$(error GIT_USERNAME is undefined)
+endif
+	# MLOps pipieline setup
 	@sed -e "s#{{ AWS_SECRET_ACCESS_KEY }}#${AWS_SECRET_ACCESS_KEY}#g" -e "s#{{ AWS_ACCESS_KEY_ID }}#${AWS_ACCESS_KEY_ID}#g" -e "s#{{ S3_REGION }}#${S3_REGION}#g" -e "s#{{ S3_ENDPOINT }}#${S3_ENDPOINT}#g" pipelines/tekton/aiedge-e2e/templates/credentials-s3.secret.yaml.template | oc create -f -
 	@sed -e "s#{{ IMAGE_REGISTRY_USERNAME }}#${IMAGE_REGISTRY_USERNAME}#g" -e "s#{{ IMAGE_REGISTRY_PASSWORD }}#${IMAGE_REGISTRY_PASSWORD}#g" pipelines/tekton/aiedge-e2e/templates/credentials-image-registry.secret.yaml.template | oc create -f -
 	oc secret link pipeline credentials-image-registry
 
+	# GITOps pipeline setup
+	sed -e "s#{username}#${GIT_USERNAME}#g" -e "s#{github_pat_1234567890ABCDAPI_TOKEN}#${GIT_TOKEN}#g" pipelines/tekton/gitops-update-pipeline/example-pipelineruns/example-git-credentials-secret.yaml | oc create -f -
```

Comment:
It might be more convenient to have the tail of this be `oc apply` rather than `oc create`, same with the ones for the build pipeline above it. WDYT? I've been bitten by that a couple of times where I tried to run it again for some reason, and it failed because you can't "create" something that already exists.

Reply:
Oh I didn't actually know that was the difference between them. Because these are in the `go-test-setup` it is expected that you only run this once for a namespace and then `go-test` as much as you want.

I will actually update the docs to say this as it wasn't there, but I will also change it to `oc apply`.